setup.cfg

```diff
@@ -26,16 +26,16 @@ classifiers =
 
 [options]
 include_package_data = True
-python_requires = >=3.9,<3.11
+python_requires = >=3.9
 cmdclass =
   install = install.AWEInstall
 install_requires =
-  AWE_Lexica @ git+https://github.com/ETS-Next-Gen/AWE_Lexica.git
-  spacy>=3.1.0,<3.4.0
+  awe_lexica
+  spacy
   holmes_extractor
```

Comment:
There was mention about no longer needing this. Is there a reason `holmes_extractor` is still around?

Reply:
for the time being, it is -- I am currently working on a different branch which works without holmes.

Reply:
Removing Holmes requires a bigger lift and new features so we will separate that from the 3.11 update.